src/common/system.rs

```diff
@@ -1929,6 +1948,7 @@ It will retrieve the following information:
     );
     impl_get_set!(ProcessRefreshKind, cmd, with_cmd, without_cmd, UpdateKind);
     impl_get_set!(ProcessRefreshKind, exe, with_exe, without_exe, UpdateKind);
+    impl_get_set!(ProcessRefreshKind, thread, with_thread, without_thread);
```

Comment:
Thinking about it some more, I think it should be named `tasks` (and the `with_` equivalents). After it's tasks, not threads.

Reply:
Yea I agree. I was thinking the same too. Proc fs does call it task